### ISSUE TITLE
Load model safetensors completely offline

### ISSUE THREAD

**Issue by MahmoudGFX** [huggingface/diffusers#4888]
### Describe the bug

the issue in 
Load model safetensors completely offline
I see that `FromSingleFileMixin.from_single_file` does not pars `original_config_file`
to pass it to  `download_from_original_stable_diffusion_ckpt`

### Reproduction

load any safetensors model without any internet 

### Logs

```shell
urllib3.exceptions.MaxRetryError: HTTPSConnectionPool(host='raw.githubusercontent.com', port=443): Max retries exceeded with url: /CompVis/stable-diffusion/main/configs/stable-diffusion/v1-inference.yaml (Caused by NewConnectionError('<urllib3.connection.HTTPSConnection object at 0x7f2728e11a00>: Failed to establish a new connection: [Errno 101] Network is unreachable'))
```

### System Info

- `diffusers` version: 0.20.2
- Platform: Windows-10-10.0.22621-SP0
- Python version: 3.9.5
- PyTorch version (GPU?): 2.0.1+cu118 (True)
- Huggingface_hub version: 0.16.4
- Transformers version: 4.31.0
- Accelerate version: 0.21.0
- xFormers version: 0.0.20
- Using GPU in script?: <fill in>

### Who can help?

@patrickvonplaten, @sayakpaul, and @williamberman

**Comment by williamberman** [comment]
hey @MahmoudGFX could we have a full code snippet to reproduce the error :)

**Comment by MahmoudGFX** [comment]
I saw this ticket and wasn't sure if the lack of internet was intentional. 
I was planning to request it as a feature. But I managed to make it work with a super small update.
So I load the safetensors like this, for example, `./` where my local file is, I pass the full  path
```
StableDiffusionXLPipeline.from_single_file('./sd_xl_base_1.0.safetensors', original_config_file='./sd_xl_base.yaml',
                                               cache_dir='./')
```

all that I have to update is  in here 
`diffusers.loaders.FromSingleFileMixin.from_single_file`
I get the configFile
`original_config_file= kwargs.pop("original_config_file", None)`
then I pass it here in line 1922 in loaders.py 
`download_from_original_stable_diffusion_ckpt`

because in `convert_from_ckpt.py` if the orginal_config_file is None, you just fetch it from the internet.
